src/main/java/com/school/school/model/ExamRating.java

```diff
@@ -0,0 +1,29 @@
+package com.school.school.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode(of = "id")
```

Comment:
Nie do konca jest to poprawny equals zastanow sie co w sytuacji kiedy id = null <- jeszcze przed dodaniem encji